pkg/genericapiserver/genericapiserver.go

```diff
@@ -429,6 +430,43 @@ func (s *GenericAPIServer) newAPIGroupVersion(apiGroupInfo *APIGroupInfo, groupV
 	}, nil
 }
 
+// TODO: Remove in 1.6. Returns if kubectl is older than v1.5.0
+func isOldKubectl(userAgent string) bool {
```

Comment:
Not in genericapiserver, please. This only applies to the master, so make the surface area here as small as possible